Tasks/VsTest/vstest.ts

```diff
@@ -874,9 +913,14 @@ function responseContainsNoTests(filePath: string): Q.Promise<boolean> {
     return utils.Helper.readFileContents(filePath, 'utf-8').then(function (resp) {
         if (resp === '/Tests:"' || resp === '/Tests:' || resp === '/TestCaseFilter:') {
             return true;
-        }
-        else {
+        } else {
             return false;
         }
     });
+}
+
+function publishEventToCi(areaCode : string, message: string) {
```

Comment:
const taskProps = { areaCode: '', message: '', tracePoint: '0', isUserError: 'false'};

Reply:
done

---
In reply to: [136253659](https://github.com/Microsoft/vsts-tasks/pull/5156#discussion_r136253659) [](ancestors = 136253659)